scenes/world_map/frays_end.tscn

```diff
@@ -884,6 +887,38 @@ hint_node = NodePath("../../ScreenOverlay/MovementInputHints")
 position = Vector2(552, 112.5)
 shape = SubResource("RectangleShape2D_h30yx")
 
+[node name="NdCrochenthemum01" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(906, -102)
+texture = ExtResource("40_06x6x")
+
+[node name="NdCrochenthemum02" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(874, -45)
+texture = ExtResource("41_xt3dw")
+
+[node name="NdCrochenthemum03" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(884, -4)
+texture = ExtResource("40_06x6x")
+
+[node name="NdCrochenthemum04" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(899, -141)
+texture = ExtResource("40_06x6x")
+
+[node name="NdMushroomBlue01" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(921, -29)
+texture = ExtResource("42_lftgk")
+
+[node name="NdMushroomBlue02" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(846, -109)
+texture = ExtResource("42_lftgk")
+
+[node name="NdMushroomBlue03" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(866, -96)
+texture = ExtResource("42_lftgk")
+
+[node name="NdCrochenthemum06" type="Sprite2D" parent="Tutorial/MovementInputHint"]
+position = Vector2(834, -85)
+texture = ExtResource("40_06x6x")
```

Comment:
You've placed these nodes inside the input hint. This means they will fade out when the hint disappears. I would instead put them inside the `Props` node (just below the `Player`).